.github/workflows/ci.yml

```diff
@@ -1,6 +1,13 @@
 name: Keycloak CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches-ignore: [main]
+  pull_request:
+    # paths-ignore:
```

Comment:
This is an initial proposal for @DGuhr suggestion, I can remove it without any problem.
Leaving it commented since it might require some "Required jobs for merging PRs" adjustments.